src/app/components/ManualEntry.svelte

```diff
@@ -1,9 +1,57 @@
 <script lang="ts">
-  /**
-   * Handles the click when the delete button is pressed
-   * TODO
-   */
-  const deleteRow = (): void => {};
+  import Button from "./Button.svelte";
+  import worker from "../spawn-worker";
+
+  interface rowDataObj {
```

Comment:
Interface names should be in PascalCase and should avoid the words `obj` and `interface` in the name. How about `DictionaryEntry`?

```suggestion
  interface DictionaryEntry {
```

(See: https://github.com/airbnb/javascript#naming--PascalCase)

Reply:
Sure, it makes more sense